### ISSUE TITLE
bring back kawaii mode

### ISSUE THREAD

**Issue by ghost** [morpho-app/Morpho#39]
![tumblr_069a6380df7fcb60cf9e154bdc5d95cb_449be626_250](https://github.com/user-attachments/assets/e798d736-4df4-4fcc-b5fd-d4169f00a743)

**Comment by orual** [comment]
![image](https://github.com/user-attachments/assets/c3be6e24-077f-43b2-9fb2-d3b9cee82263)

**Comment by ghost** [comment]
hmm need to figure out how to optimize the image rendering

**Comment by orual** [comment]
yeah. need to check on mobile to see if this is a problem with skiko specifically (the image rendering backend on desktop) or just in general.

**Comment by orual** [comment]
i've had a hell of a time with the images in the posts on desktop looking like garbage when scaled without a bunch of fiddling, but they mostly look fine on mobile.

**Comment by orual** [comment]
![image](https://github.com/user-attachments/assets/26510b77-edef-4ffc-895e-d98bf15f03af)
Was adding an icon to open the nav drawer where I'm putting access to the settings menu, etc., decided to have this option. Still need to solve some image scaling, but...
